sigstore/_internal/fulcio/client.py

```diff
@@ -262,14 +263,14 @@ def _opaque16(value: bytes) -> bytes:
             raw_sct = _opaque16(_opaque16(raw_sct_list_bytes))
 
             sct = precert_scts_extension[0]
-        except ExtensionNotFound:
+        else:
             # If we don't have any embedded SCTs, then we might be dealing
             # with a Fulcio instance that provides detached SCTs.
 
             # The SCT header is a base64-encoded payload, which in turn
             # is a JSON representation of the SignedCertificateTimestamp
             # in RFC 6962 (subsec. 3.2).
-            sct_b64 = resp.headers.get("SCT")
+            sct_b64 = resp.json()["signedCertificateDetachedSct"]["signedCertificateTimestamp"]
```

Comment:
I think we need to use `get` in order for the check below to work. I expect that this will just raise a `KeyError`.

Reply:
Changed, thanks!